examples/demo_actor/README.md

```diff
@@ -36,8 +36,8 @@ export PYTHONPATH=`pwd`
 1. Run Demo Actor service in new terminal window
 
 ```bash
-$ cd examples/demo_actor
-$ dapr run --app-id demo-actor --app-port 3000 -- uvicorn --dapr-http-port 3000 demo_actor_service:app
+$ cd ./demo_actor
```

Comment:
can we remove this

Reply:
nope demo_actor has demo_actor sub-directory.